test/b2gedmntuples_cfg.py

```diff
@@ -749,9 +749,24 @@
     candSVTagInfos         = cms.string("pfInclusiveSecondaryVertexFinder"), 
 )
 
+
+process.photonJetsUserData = cms.EDProducer(
+    'PhotonJets',
+    phoLabel = cms.InputTag("slimmedPhotons"),
+    pv        = cms.InputTag(pvLabel),
+    rho               = cms.InputTag(rhoLabel),
+    packedPFCands = cms.InputTag("packedPFCandidates"),
+    jetLabel  = cms.InputTag("jetUserDataAK8"),
+    ebReducedRecHitCollection = cms.InputTag("reducedEgamma:reducedEBRecHits"),
+    eeReducedRecHitCollection = cms.InputTag("reducedEgamma:reducedEERecHits")
+
+    )
+
+
+
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
-    jetLabel  = cms.InputTag('jetUserDataAK8'),
+    jetLabel  = cms.InputTag('photonJetsUserData'),
     #topjetLabel = cms.InputTag('patJetsCMSTopTagCHSPacked'),
```

Comment:
Hi Savvas, why is the jet label here replaced?
Also, please refresh my memory: what is the photonJets collection supposed to have?